docs/examples/django/README.rst

```diff
@@ -0,0 +1,147 @@
+OpenTelemetry Django Instrumentation Example
+============================================
+
+This shows how to use `opentelemetry-ext-django` to automatically instrument a
+Django app.
+
+For more user convenience, a Django app is already provided in this directory.
+
+Preparation
+-----------
+
+This example will be executed in a separate virtual environment:
+
+.. code-block::
+
+    $ mkdir django_auto_instrumentation
```

Comment:
I think the examples in the docs should show programmatic instrumentation instead of automatic. It is difficult to tell which parts belong to auto-instrumentation and which parts are part of `opentelemetry-ext-django`.

Reply:
Fixed